README.md

```diff
@@ -115,13 +115,15 @@ OPTIONS:
 
 SUBCOMMANDS:
     add             Adds a new pin entry
+    freeze          Freeze a pin entry
     help            Prints this message or the help of the given subcommand(s)
     import-flake    Try to import entries from flake.lock
     import-niv      Try to import entries from Niv
     init            Intializes the npins directory. Running this multiple times will restore/upgrade the
                     `default.nix` and never touch your sources.json
     remove          Removes one pin entry
     show            Lists the current pin entries
+    thaw            Thaw a pin entry
```

Comment:
I get that logically `freeze` <-> `thaw` makes sense but from a UX perspective I would prefer to have a more explicit indication that those two commands belong together.

Hence I'd prefer `freeze` and `unfreeze` - WDYT @andir  ?

Reply:
I'd also prefer `unfreeze`, with an optional alias to `thaw` for those who find that more intuitive